examples/escrow/tests/escrow.ts

```diff
@@ -1,20 +1,22 @@
-const anchor = require("@project-serum/anchor");
-const { TOKEN_PROGRAM_ID, Token } = require("@solana/spl-token");
-const assert = require("assert");
+import * as anchor from "@project-serum/anchor";
+import { TOKEN_PROGRAM_ID, Token } from "@solana/spl-token";
+import {assert} from "chai";
+
+import {EscrowIDL} from "./types/escrow";
 
 describe("escrow", () => {
   const provider = anchor.Provider.env();
   anchor.setProvider(provider);
 
-  const program = anchor.workspace.Escrow;
+  const program = anchor.workspace.Escrow as anchor.Program<EscrowIDL>;
```

Comment:
Hmm the account namespace doesn't type check. I can do `program.account.notAValidType` and it compiles.

Reply:
Is this expected @macalinao or did I miss something.

Reply:
I may have forgotten to add this in

Reply:
I fixed this by [tweaking](https://github.com/project-serum/anchor/pull/537/files#diff-0e2c48e520eafaabcd0d9de371c7d2e74f5bbb4eba766216bb24efb265c0e5c3R68) the `AccountNamespace`. But for this to work, I also needed to customize the script that [produces](https://github.com/project-serum/anchor/pull/537/files#diff-9db4b8fb68cb3e81dd2ec7578b892eafc3ba7eb1db7535ecd5d41a76fee7fdbaR13) the idl type such that the Account names are camel case.

I think we'd want to think through the dev flow for producing the IDL type definition if we are to ship this. Maybe we add a script to `anchor-cli` that produces the IDL types for you?

Reply:
Putting this on your radar @armaniferrante

Reply:
Is this ready for another review/testing?